lib/studies/reply/compose_page.dart

```diff
@@ -0,0 +1,281 @@
+
+  @override
+  __SenderAddressRowState createState() => __SenderAddressRowState();
+}
+
+class __SenderAddressRowState extends State<_SenderAddressRow> {
+  String senderEmail;
+
+  @override
+  void initState() {
+    super.initState();
+    senderEmail = widget.senderEmail;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final textTheme = theme.textTheme;
+    final accounts = [
+      'flutterfan@gmail.com',
+      'materialfan@gmail.com',
+    ];
+
+    return PopupMenuButton<String>(
+      padding: EdgeInsets.zero,
+      onSelected: (email) {
+        setState(() {
+          senderEmail = email;
+        });
+      },
+      itemBuilder: (context) => <PopupMenuItem<String>>[
+        PopupMenuItem<String>(
+          value: accounts[0],
+          child: Text(
+            accounts[0],
+            style: textTheme.bodyText2,
+          ),
+        ),
+        PopupMenuItem<String>(
+          value: accounts[1],
+          child: Text(
+            accounts[1],
+            style: textTheme.bodyText2,
+          ),
+        ),
+      ],
+      child: Padding(
+        padding: const EdgeInsets.only(
+          left: 12,
+          top: 16,
+          right: 10,
+          bottom: 10,
+        ),
+        child: Row(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Expanded(
+              child: Text(
+                senderEmail,
+                style: textTheme.bodyText2,
+              ),
+            ),
+            Icon(
+              Icons.arrow_drop_down,
+              color: theme.colorScheme.onSurface,
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _RecipientsRow extends StatelessWidget {
+  const _RecipientsRow({@required this.recipients, @required this.avatar})
+      : assert(recipients != null),
+        assert(avatar != null);
+
+  final String recipients;
+  final String avatar;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 4, horizontal: 12),
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          Expanded(
+            child: Wrap(
+              children: [
+                Chip(
+                  backgroundColor:
+                      Theme.of(context).chipTheme.secondarySelectedColor,
+                  padding: EdgeInsets.zero,
+                  avatar: CircleAvatar(
+                    backgroundImage: AssetImage(
+                      '$avatar',
+                      package: 'flutter_gallery_assets',
+                    ),
+                  ),
+                  label: Text(
+                    recipients,
+                  ),
+                ),
+              ],
+            ),
+          ),
+          InkResponse(
+            customBorder: const CircleBorder(),
+            child: const Icon(Icons.add_circle_outline),
```

Comment:
should it always be black?

Reply:
No, but it changes automatically based on the theme. White on dark mode and black on light mode.